server/pages/desktop.jade

```diff
@@ -26,6 +26,23 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class='is-desktop' + ( isFluidWidth ?
 		script(src='/calypso/build-desktop.js')
 		script(src='/desktop/desktop-app.js')
 
+		script.
```

Comment:
this seems really strange here: using the dot on the function itself and then the continuation of the dot is an IIFE

Could we rearrange this somehow?

I'm not sure how Jade handles things in this context, but I'm pretty sure we could take several approaches:

 - Somehow rearrange the function here
 - Pass in some variable which Jade uses whereas the user agent is detected outside of Jade
 - Some kind of mixin or template from outside?